controllers/AbstractApiController.php

```diff
@@ -246,20 +249,20 @@ public function addDataset(string $name, $project_id, $manager_id,
      * @throws Exception
      */
     public function updateDataset($id, string $name, $project_id, $manager_id,
+                                  int $country_id,
                                   int $type = App::DATASET_TYPE_UNKNOWN,
                                   string $assessment_date = null,
                                   string $description = ''): object
     {
         $datasetController = new DatasetsController($this->getAuth());
         return $datasetController->patchResource($id,
-            array(
-                'name' => $name,
+                ['name' => $name,
                 'project_id' => $project_id,
                 'manager_id' => $manager_id,
                 'type_id' => $type,
                 'description' => $description,
-                'assessment_date' => $assessment_date
-            ));
+                'assessment_date' => $assessment_date,
+                'country_id' => $countryId]);
```

Comment:
`Undefined variable '$countryId'`

Reply:
Argh. I don't see this on mine at all, trying to work out what I've done....

Reply:
Hahahahahahahahahaha looking at wrong method, keep forgetting there's two of them